test/rfc8032-test-vectors.h

```diff
@@ -0,0 +1,779 @@
+#ifndef RFC8032_TEST_VECTORS_H
```

Comment:
Why didnt you use evp_test.c?

Reply:
I'm not sure I understand.

Are you asking why I chose to add the rfc8032 test vectors in a new header file rather than just add them directly inside `evp_extra_test.c` ?

It just didn't occur to me to do that.  There is quite a lot of test vector data (the header file is over  700 lines).  Keeping them in a separate header seemed reasonable.

Reply:
This question was - why did you not use evp_test.c.
evp_test.c loads files from test/recipes/30-test_evp_data/*.txt. 
A lot of algorithms are tested this way. i.e Instead of being included from a header does it make sense to add this to a txt file and use evp_test.c

Reply:
Would it be too much to ask to convert these testcases to evp_test data testcases?

It would require modification of evp_test.c to pass the algorithm variant as params though.

Reply:
Hi Tomas.  Refactoring the test and reformatting all the test vector data again sounds really fun :wink: 

Seriously though, I don't mind changing it, but I would prefer to do that in a separate PR.